main-app/app/scripts/game/services/game-api.js

```diff
@@ -3,10 +3,24 @@
     angular.module('Tombola.Games.Bingo90.Core')
         .service('GameApi',  ['GameProxy', function (gameProxy) {
```

Comment:
I'd split into lobby service and game service? 
Not sure - you'd have to make a call on that.